server/services/facets/saved_objects.ts

```diff
@@ -0,0 +1,154 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+export default class SavedObjectFacet {
+  constructor(private client: any) {
+    this.client = client;
+  }
+
+  fetch = async (
+    request: any,
+    format: string,
+    responseFormat: string,
```

Comment:
can `responseFormat` be different in the future? doesn't seem to be used

Reply:
I can remove this.